recipe/build.sh

```diff
@@ -8,12 +8,13 @@ then
     export CC=clang
     export CXX=clang++
     export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
-    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11 -O2"
     export LIBS="-lc++"
 elif [ "$(uname)" == "Linux" ];
 then
     export CC=gcc
     export CXX=g++
+    export CXXFLAGS="-O2"
```

Comment:
Probably better to move the `-O2` flags addition below. Also it would be good to make sure existing things in `CXXFLAGS` are not dropped in the process.